packages/twenty-server/src/main.ts

```diff
@@ -54,6 +58,18 @@ const bootstrap = async () => {
   app.useGlobalPipes(
     new ValidationPipe({
       transform: true,
+      exceptionFactory: (validationErrors: ValidationErrorType[] = []) => {
```

Comment:
I'm a bit lost, this is the pipe which is validating our graphql input types !?

Nest is so weird, I would expect graphql yoga to expose it instead
@Weiko what do you think?

Reply:
@charlesBochet I think it's validating class-validator globally, whether it's from typeorm or yoga or any internal part that's using class-validator. Ideally we would want to use an ExceptionFilter class / or check if we can make it work with useGraphQLErrorHandlerHook hook for yoga 🤔